### ISSUE TITLE
Start with Sega - Saturn/ST-V (Kronos) core failed, missing libGL.so.1

### ISSUE THREAD

**Issue by VergilGao** [spleen1981/retroarch-kodi-addon-CoreELEC#20]
```
[INFO] [Joypad]: Found joypad driver: "udev".
[INFO] [Video]: Found display server: "null".
[WARN] [udev]: Mouse grab/ungrab feature unavailable.
[INFO] [ALSA]: Using signed 16-bit format.
[INFO] [ALSA]: Period size: 768 frames
[INFO] [ALSA]: Buffer size: 3072 frames
[INFO] [ALSA]: Can pause: yes.
[INFO] [Display]: Found display driver: "gl".
[ERROR] 打开libretro核心失败: "/storage/.kodi/addons/script.retroarch.launcher.Amlogic-ng.arm/lib/libretro/kronos_libretro.so"
[ERROR] Error(s): libGL.so.1: cannot open shared object file: No such file or directory
```

**Comment by spleen1981** [comment]
fixed by 20be7fad8b80ff61bf3d87bf7a7e34d31cb6b5fd

**Comment by spleen1981** [comment]
However this core will not work for Amlogic-ng, as GLcore >= 4.2 is required.
It will be dropped in next release, bc76c2cc5e2f32d27ecfee91f7d3b314d5abdb92
